doc/devel/contribute.rst

```diff
@@ -287,7 +285,7 @@ guide you through each step:
 4. Check existing pull requests (e.g., :ghpull:`28476`) and filter by the issue number to make sure the issue is not in progress:
 
    * If the issue has a pull request (is in progress), tag the user working on the issue, and ask to collaborate (optional).
-   * If a pull request does not exist, create a `draft pull request <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests>`_ and follow the `pull request guidelines <https://matplotlib.org/devdocs/devel/pr_guide.html>`_.
+   * If a pull request does not exist, create a :ref:`draft pull request <draft-pr>`.
```

Comment:
I had missed this.  My reading of this is that we encourage new contributors to open in draft whether the work is merge-ready or not.

It also doesn’t really make sense to me to go straight from "choose an issue" to "open the [draft] PR".  Maybe this should instead link to "Start a pull request " at the bottom of this page, which links out to the development setup instructions, etc.

Of course, you may consider this comment out-of-scope for the current change.

Reply:
Done.